arch/risc-v/Kconfig

```diff
@@ -188,6 +189,30 @@ config ARCH_MMU_TYPE_SV39
 	bool
 	default n
 
+config ARCH_HAVE_S_MODE
```

Comment:
Should we move this to common place ?

Reply:
There is no more common place for this than arch/risc-v/Kconfig. S_MODE is RISC-V iSA specific thing. What do you mean?

Reply:
Please ignore this , we can do that if other chip (like arm based) support supervisor in future